trading_calendars/exchange_calendar_xasx.py

```diff
@@ -40,18 +40,28 @@
 
 NewYearsDay = new_years_day(observance=weekend_to_monday)
 
+# Jevons Global: the present observance rule follows the 1994 agreement between the States to synchronize the holiday
 AustraliaDay = Holiday(
     'Australia Day',
     month=1,
     day=26,
+    start_date=Timestamp('1994-01-01'),
     observance=weekend_to_monday,
 )
 
+#  Jevons Global: note that prior to 1993 the holiday was observed on the Monday following, or on, the 26th of January
```

Comment:
I don't know how you were able to get this information from so long ago, but thank you!

Reply:
@jmccorriston a range of sources including newspaper checks from an online archive. Let me see if I can codify those. The US presidents holidays came from an NYSE publication. I will go back and find url links where possible.

Reply:
On those fixes to tests and the like, give me a day or so to work it out.

Reply:
@savvyyabby I'm impressed! I have found it extremely difficult to find corroborating sources prior to ~1990/2000 when it became more common to publish this type of news online.

Reply:
@jmccorriston This online archive at the Australian National Library helped me... The Canberra Times published daily stock notices from the Sydney markets. I used that to check some of the more bizarre hiccups in how Australia Day and others were treated over time. It is tricky in Oz because each capital city used to have their own market and they had different holiday dates. That is why these digital newspaper archives are helpful. It is just the whacky stuff, like Australia Day, some ANZAC days and Bicentennial Holidays or specials like Y2K or when NYE is on a Monday .... or something like that. https://trove.nla.gov.au/newspaper/title/11

Reply:
I did a bit on India too... there are some good resources from the Reserve Bank of India. I will dig out those too...

Reply:
Example of what I mean from newspaper... Sydney stock exchange industrials Jan 5th 1970.

[https://trove.nla.gov.au/newspaper/article/131681315?browse=ndp%3Abrowse%2Ftitle%2FC%2Ftitle%2F11%2F1970%2F01%2F09%2Fpage%2F14540107%2Farticle%2F131681315]

I would just check the week in which a holiday was due to see the observance, like whether the holiday had been mondayized or not.

Reply:
Wow, that's awesome!